extensions/2.0/Khronos/KHR_spz_gaussian_splats_compression/README.md

```diff
@@ -0,0 +1,144 @@
+
+## Compressing 3D Gaussian splats using SPZ
+
+If a primitive contains an `extension` property which defines both `KHR_gaussian_splatting` and `KHR_spz_gaussian_splats_compression` then support for SPZ compression is required. There is no requirement for a backup uncompressed buffer.
+
+The extension must be listed in `extensionsUsed` alongside `KHR_gaussian_splatting`.
+
+```json
+  "extensionsUsed" : [
+    "KHR_gaussian_splatting",
+    "KHR_spz_gaussian_splats_compression"
+  ]
+```
+
+It must also be listed in `extensionsRequired`. When `KHR_spz_gaussian_splats_compression` is in use, the `KHR_gaussian_splatting` extension must also be listed as required.
+
+```json
+  "extensionsRequired" : [
+    "KHR_gaussian_splatting",
+    "KHR_spz_gaussian_splats_compression"
+  ]
+```
+
+### Inheritance from base extension
+
+As this extension extends the base extension, all components of the base extension are inherited. Any modifications to the base definition are explicitly defined in the sections below.
+
+### Schema Example
+
+Example SPZ extension shown below. This extension only affects any `primitive` nodes containting Gaussian splat data. Note that unlike the base `KHR_gaussian_splatting` extension, the `indices` property is excluded, and a `bufferView` is provided by the extension. This bufferview points to where the SPZ blob is stored.
+
+```json
+  "meshes": [{
+      "primitives": [{
+          "attributes": {
+            "POSITION": 0,
+            "COLOR_0": 1,
+            "_SCALE": 2,
+            "_ROTATION": 3,
+            "_SH_DEGREE_1_COEF_0": 4,
+            "_SH_DEGREE_1_COEF_1": 5,
+            "_SH_DEGREE_1_COEF_2": 6
+          },
+          "material": 0,
+          "mode": 0,
+          "extensions": {
+            "KHR_spz_gaussian_splats_compression": {
+              "bufferView": 0,
+            }
+          }
+        }]
+    }],
+  "buffers": [{
+      "uri": "0.bin",
+      "byteLength": 9753142
+    }],
+  "bufferViews": [{
+      "buffer": 0,
+      "byteLength": 9753142
+    }],
+```
+
+### Extension Properties
+
+#### bufferView
+
+This property points to the bufferView containing the Gaussian splat data compressed with SPZ.
+
+### Accessors
+
+Accessor requirements are modified from the base `KHR_gaussian_splatting` extension with the following adjustments to definition:
+
+ - SPZ compressed attributes must not include `bufferView` nor `byteOffset`. (See: [Conformance](#conformance))
+ - Accessor `type` is defined for the resulting type after decompression and dequantization has occurred.
+ - The accessor `count` must match the number of points in the compressed SPZ data.
+

```

Comment:
There should be explicit language for matching attribute semantic names defined in `KHR_gaussian_splatting` to data structures defined in SPZ specs.